managed_config/index.bs

```diff
@@ -0,0 +1,98 @@
+<pre class='metadata'>
+Title: Managed Configuration API
+Shortname: managed-configuration
+Level: 1
+Editor: Anatoliy Potapchuk (Google) apotapchuk@chromium.org
+Status: w3c/WD
+Issue Tracking: Github https://github.com/WICG/WebApiDevice/issues
+URL: http://example.com/url-this-spec-will-live-at
+Abstract: This document defines a web platform API that enables developers to access administrator-set configuration on managed devices. Such API invaluable for such use cases as digital signage and device-centric personalization.
+TR: 
+</pre>
+
+<pre class=link-defaults>
+spec:html; type:interface; text:Navigator
+</pre>
+
+<pre class="anchors">
+
+</pre>
+
+# Model # {#model}
+
+## Managed devices ## {#managed-devices}
+
+The API is presumed to be used on devices, which are not fully controlled by the end user, but rather by an external entity, the device administrator. Device administrators are given the power to fully control the managed device.
+
+## Managed configuration ## {#managed-configuration}
+
+For the purposes of this specification the device administrator can control a list of managed web applications. Each entry in this list can have a set JSON configuration, which is accessible by the documents hosted at the origin of the web application.
+
+Note that this does not apply to anonymous contexts, where the observed state is indistinguishable from a non-managed one.
+
+## <dfn dfn>Data model</dfn> ## {#data-model-section}
+
+We can assume without loss of generality that managed configuration is stored in a two-level key-value store. The first level of this store corresponds to configured origins, while the second corresponds to per-origin configuration string keys.
+## Data integrity verification ## {#data-integrity-verification}
+
+There is no managed configuration verification mechanism defined in the specification, which implies that the website should assume that the configuration could be tampered with, stolen or be replayed on a non-managed device. 
+
+Alternative security measures should be put in place by the website itself.
+
+# Extensions to the {{Navigator}} interface # {#navigator-extensions}
+
+<xmp class="idl">
+[
+  SecureContext
+] partial interface Navigator {
+  [SecureContext, SameObject]
+  readonly attribute NavigatorManagedData managed;
+};
+</xmp>
+
+
+{{managed}} attribute
+When getting, the managed attribute always returns the same instance of the {{NavigatorManagedData}} object.
+
+# {{NavigatorManagedData}} interface # {#navigatormanageddata-interface}
+
+<xmp class="idl">
+[
+  SecureContext,
+  Exposed=Window
+] interface NavigatorManagedData : EventTarget {
+  // Managed Configuration API.
+  Promise<object> getManagedConfiguration(sequence<DOMString> keys);
```

Comment:
This should return `record<DOMString, DOMString>`. See https://heycam.github.io/webidl/#es-record for an example of how to construct a new instance of `record<DOMString, DOMString>` key by key.

Reply:
the value does not have to be a DOMString